swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
Everything we add has a maintenance cost, regardless of size. It provides no useful function to the iPaaS itself and if it did I would want it deployed as a separate unit anyway, not embedded in here, following the single responsibility principle.

Regardless, I don't see this question blocking this reasonably large change migrating to Spring Boot. If people really feel strongly about it, then open an issue to discuss, but please don't block this PR due to swagger UI integration when there are a myriad of ways to use swagger ui, the simplest being the chrome extension I mentioned above.